libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -226,6 +242,19 @@ void AP_GPS::init(DataFlash_Class *dataflash, const AP_SerialManager& serial_man
     _port[0] = serial_manager.find_serial(AP_SerialManager::SerialProtocol_GPS, 0);
     _port[1] = serial_manager.find_serial(AP_SerialManager::SerialProtocol_GPS, 1);
     _last_instance_swap_ms = 0;
+
+    // Initialise class variables used to do GPS blending
+    memset(&_NE_pos_offset_m, 0, sizeof(_NE_pos_offset_m));
```

Comment:
They are declared, but not initialised elsewhere. I am following the method of initialisation used previously for other class variables.

I can't comment on the flash issue.